pages/index.js

```diff
@@ -56,7 +56,7 @@ const TwitterButton = ({ universalLink }) => {
   return (
     <TwitterShareButton
       url={universalLink}
-      message={intl.formatMessage({id: 'Share.Twitter.Tweet', defaultMessage: 'Run OONI Probe to test for censorship!'})}
+      message={intl.formatMessage({ id: 'Share.Twitter.Tweet', defaultMessage: 'Run OONI Probe to test for censorship!' })}
```

Comment:
It looks like my editor has done some automatic reformatting of several blocks. Most changes can be ignored.

Reply:
Thanks!